masonry/src/event_loop_runner.rs

```diff
@@ -284,6 +284,46 @@ impl ApplicationHandler<accesskit_winit::Event> for MainState<'_> {
                         self.pointer_state.clone(),
                     ));
             }
+            WinitWindowEvent::Touch(winit::event::Touch {
+                location, phase, ..
+            }) => {
+                // FIXME: This is naïve and should be refined for actual use.
```

Comment:
When this lands, we should add a comment in #312 pointing at this section of code (as an example of what not to do lol)

Reply:
I mean... it is ‘Unified’ :upside_down_face: